mesheryctl/pkg/utils/helpers.go

```diff
@@ -63,6 +63,8 @@ const (
 var (
 	// ResetFlag indicates if a reset is required
 	ResetFlag bool
+	// MesheryEndpoint is the default URL in which Meshery is exposed
+	MesheryEndpoint = "http://localhost:9081"
```

Comment:
Isn’t this to be loaded from context?

Reply:
@leecalcote This would be the default url when creating a new context. So I moved it to a global variable. I thought that was what you meant for me to do.

Reply:
Oh, boy. I forgot that this PR is making context command itself. Please disregard my last comment. 😬